Source/WebCore/SaferCPPExpectations/UncountedCallArgsCheckerExpectations

```diff
@@ -1313,6 +1312,7 @@ style/StyleBuilder.cpp
 style/StyleBuilderConverter.h
 style/StyleBuilderCustom.h
 style/StyleBuilderState.cpp
+style/StyleInterpolation.cpp
```

Comment:
Would you mind adding StyleInterpolationFunctions.h and StyleInterpolationWrappers.h here as well?
That'll keep the safer C++ bot green.